data/core/units/undead/Corpse_Soulless.cfg

```diff
@@ -146,9 +146,13 @@
         inherit=yes
         profile=unit_image
         {UNIT_BODY_SOULLESS_STATS    fly 5 33}
-        {UNIT_BODY_SOULLESS_GRAPHICS soulless-drake 3}
```

Comment:
I thought I commend this a bit more detailed:
This is the only place where the value of that line was not 4. Since the update of the soulless drake images earlier in 1.13 it are 4 images in this case too. So, since in this, as well as in the walking corpse file, this is everywhere 4, I simplified the macro to use one argument less and the death animation knows that it are every time 4 frames.